tests/lean/run/437.lean

```diff
@@ -0,0 +1,6 @@
+example (a : ℕ) : true :=
```

Comment:
I think we've been trying to name test files that reference `leanprover-community` issue numbers with a `c`, so this should be called `437c.lean`.

Reply:
Ah thanks, I wondered whether there was a convention for that, but I saw some other test files like `####b.lean`, so I didn't work out that `c` meant community.